stories/toolbar/toolbar.stories.js

```diff
@@ -115,15 +115,19 @@ export const overflow = () => `<div style="height:250px">
             <div class="fd-popover__body fd-popover__body--right fd-popover__body--no-arrow"
                 aria-hidden="false"
                 id="mw0daw8d7h">
-                <div class="fd-toolbar__overflow__body">
+                <div class="fd-toolbar__overflow">
                     <label class="fd-label fd-toolbar__overflow__label">Label</label>
-                    <button class="fd-button fd-button--compact fd-button--transparent">Edit</button>
+                    <button class="fd-button fd-button--compact fd-button--transparent fd-toolbar__button">Edit</button>
                     <span class="fd-toolbar__separator"></span>
-                    <label class="fd-form-label fd-toolbar__overflow__form-label">Form label</label>
-                    <button class="fd-button fd-button--compact fd-button--transparent">Delete</button>
+                    <label class="fd-form-label fd-toolbar__overflow__label">Form label</label>
```

Comment:
`fd-toolbar__overflow__label` is not a valid BEM

Reply:
Modified.